frontend/src/features/properties/map/activity/detail/ActivityForm.tsx

```diff
@@ -45,13 +50,41 @@ export const ActivityForm: React.FunctionComponent<IActivityFormProps> = ({
       });
     }
   };
+  const EditForm = formContent?.EditForm;
+  const ViewForm = formContent?.ViewForm;
+  const initialValues = ActivityModel.fromApi(activity, file.fileType);
+  if (
+    !!initialValues.activityData?.version &&
+    formContent !== undefined &&
+    initialValues.activityData?.version !== formContent?.version
+  ) {
```

Comment:
Did this happen too often?

Reply:
It didn't happen at all, but it could happen in the future.